CHANGELOG.next.asciidoc

```diff
@@ -34,6 +34,7 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 - Corrects issue with security events with source IP of "LOCAL" or "Unknown" failing to ingest {issue}19627[19627] {pull}34295[34295]
 - Added processing for Windows Event ID's 4797, 5379, 5380, 5381, and 5382 for the Security Ingest Pipeline {issue}34293[34293] {pull}34294[34294]
 - Added processing for Windows Event ID's 5140 and 5145 for the Security Ingest Pipeline {pull}34352[34352]
+- Handle empty DNS answer data in QueryResults for the Sysmon Pipeline {pull}35207[35207]
```

Comment:
This is an addition rather than a breaking change.

Reply:
Right, moved to the correct section.